doc/Makefile

```diff
@@ -49,17 +49,20 @@ schema.md: parse_gx_xsd.py schema_template.md ../lib/galaxy/tools/xsd/galaxy.xsd
 
 source/dev/schema.rst: schema.md ## Convert Galaxy Tool XSD Markdown docs into reStructuredText (expects pandoc in environment)
 	pandoc schema.md -f markdown_github-hard_line_breaks -s --toc --toc-depth=2 -o $@
+	sed -i -e 's|.. code:: xml|.. code-block:: xml|g' $@
 
 # might also want to do
 # cd source/lib; hg revert; rm *.rst.orig;  or not.
 clean:
 	-rm -rf $(BUILDDIR)/* $(UPDATEWORKDIR) schema.md $(GENERATED_RST)
 
-html: $(GENERATED_RST)
+_html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
+html: $(GENERATED_RST) _html
+
```

Comment:
I reverted this part - I like that pattern because it allows me to test targets one at a time but maybe there is a better way to debug Makefile stuff?